### DISCUSSION TITLE
Welcome to logo-maker Discussions!

### DISCUSSION THREAD

**Discussion by Soare-Robert-Daniel** [Codeinwp/logo-maker#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maybejustdai** [comment]
@Soare-Robert-Daniel @selul I'm having trouble setting up the PWA version on amplify. I get a 'FAILED' build error every time I try to deploy.  I've attached a screenshot for reference. Thanks for the help in advance.

- Dai 

![FireShot Capture 044 - AWS Amplify console - us-east-1 console aws amazon com](https://user-images.githubusercontent.com/110631718/200318226-c99037d3-2fbd-42b0-8988-5933037470c7.png)

**Reply by Soare-Robert-Daniel** [reply]
Hi @maybejustdai. Thanks for checking our product. Regarding the error, are you using this https://github.com/Codeinwp/logo-maker-pwa to make the build?

**Reply by maybejustdai** [reply]
Yes, I am using the PWA version of the Logo maker for this build. I also realized when downloading and extracting that other unexpected errors stop certain files from downloading.

**Reply by Soare-Robert-Daniel** [reply]
Very weird behavior. I have not touched this for almost 2 years. Some packages might be broken or incompatible. As a small curiosity, how did you discover the app? Were you using our Mobile app or just Google? Or just searching for PWA App examples?

**Reply by maybejustdai** [reply]
Sorry for any confusion, re-attempted it by cloning it and It works perfectly! Question, is it possible to add mockup previews and a checkout option at end of the process?

**Reply by maybejustdai** [reply]
I found it through the search on Github some time ago.

**Reply by Soare-Robert-Daniel** [reply]
There is no option coded for checkout (is free product). But what you mean exactly by mockup preview?

**Reply by maybejustdai** [reply]
Mockup previews to see the logo on other branding materials to show real-life use.

**Reply by Soare-Robert-Daniel** [reply]
To show the logo in the context of Intragram/Facebook/Twitter page?

**Reply by maybejustdai** [reply]
Yes, business cards, letterheads, or other stationary material.

**Reply by Soare-Robert-Daniel** [reply]
That is a very interesting idea. I haven't seen it before on other logo-maker websites. The current version does not have this feature. In the future, when we maybe revive the project, the mockup will be present. But the checkout part is tricky because it depends on the vendor (we might use Stripe, but you want PayPal, or it will be forever free™️ ). 

Thanks for the feedback 👍

**Comment by maybejustdai** [comment]
Every time I try a new build (after editing some CSS) I get this error: "There was an issue connecting to your repo provider, click "Reconnect repository" in General Settings, and then try your build again."I've reconnected in AWS Amplify and tried a new fresh build without edits (the initial build fails at the first build now, any suggestions?

**Reply by Soare-Robert-Daniel** [reply]
Never encountered it before. You can ask AWS support for clarification.
